### PR TITLE
advertiser integration  "mediaimpact"

### PR DESCRIPTION
**Pull request by adtechnology**
I signed it!

### CODE REVIEW

File: ads/_config.js [hunk 1]
```diff
@@ -53,6 +54,7 @@ export const adPreconnect = {
     'https://tpc.googlesyndication.com',
   ],
   dotandads: 'https://bal.ad.dotandad.com',
+  mediaimpact: 'https://ec-ns.sascdn.com',
```
Comment (cramforce) [inline ads/_config.js]: You can remove this. Prefetch above implies preconnect.

File: ads/mediaimpact.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    async: true
+  }
+  loadScript(global, "https://ec-ns.sascdn.com/diff/251/divscripte/amp.js?dom=" + window.context.location.host, () => {
+    if (!document.getElementById('sas_' + data.slot.replace('sas_',''))) {
+      var adContainer = document.createElement('div');
+      adContainer.id = 'sas_' + data.slot.replace('sas_','');
+      document.body.appendChild(adContainer);
+    }
+  });
+}
```
Comment (cramforce) [inline ads/mediaimpact.js]: Please add a trailing newline.

File: ads/mediaimpact.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      data-page="amp"
+      data-format="4459"
+      data-target=""
+      data-slot="4459">
+  </amp-ad>
+```
+
+## Configuration
+
+For semantics of configuration, please see ad network documentation.
```
Comment (cramforce) [inline ads/mediaimpact.md]: Could you add a link to your own docs here?

**Comment by adtechnology** [comment]
I signed it!

**Comment by adtechnology** [comment]
I signed it!

**Comment by cramforce** [comment]
Please fix lint errors and squash commits. Then this is good to go.

Thanks!

**Comment by renebaudisch** [comment]
I signed it!

**Comment by cramforce** [comment]
Please also squash commits! https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

**Comment by adtechnology** [comment]
I signed it!

File: ads/mediaimpact.md (hunk 3 above)
Comment (cramforce) [inline ads/mediaimpact.md]: Friendly ping on this

**Comment by cramforce** [comment]
CLA bot can be a little bit too strict. Thanks for your "I signed it" statements, we will definitely merge.

I pinged one more comment. We really need some way for users to go from "They support media impact" to "How do I get more info / talk to them?".

**Comment by adtechnology** [comment]
How about "for further informations  for this implementation please contact adtechnology@axelspringer.de or visit http://www.mediaimpact.de/"?

**Comment by cramforce** [comment]
+1

**Comment by adtechnology** [comment]
I signed it!

**Comment by adtechnology** [comment]
please process again, CLA is available

**Pull request merged**